pyo3-build-config/src/impl_.rs

```diff
@@ -154,30 +154,31 @@ pub struct InterpreterConfig {
 
 impl InterpreterConfig {
     #[doc(hidden)]
-    pub fn emit_pyo3_cfgs(&self) {
+    pub fn emit_pyo3_cfgs(&self, mut writer: impl Write) {
```

Comment:
Clippy appears to be unhappy about passing `stdot` manually. 

I wonder we should try to simplify this more aggressively and e.g. have this method return `String` or even `Vec<String>` which the outside code then has to be print.

Reply:
Done and force-pushed.